symplyphysics/laws/electricity/circuits/LC_oscillator.py

```diff
@@ -0,0 +1,46 @@
+from sympy import sqrt, pi
```

Comment:
Rename the file to reflect the law, eg `oscillation_period_for_capacitor_inductor_node.py`